ferry.go

```diff
@@ -383,9 +389,29 @@ func checkConnectionForBinlogFormat(db *sql.DB) error {
 
 	row = db.QueryRow("SHOW VARIABLES LIKE 'binlog_row_image'")
 	err = row.Scan(&name, &value)
+	if err != nil {
+		return err
+	}
+
 	if strings.ToUpper(value) != "FULL" {
 		return fmt.Errorf("binlog_row_image must be FULL, not %s", value)
 	}
 
 	return nil
 }
+
+func checkConnectionIsWritable(db *sql.DB) error {
```

Comment:
As a part of #35, I made all these methods under the scope of Ferry as they are (1) not used anywhere else and also are (2)  private methods such that nothing external can use it. Can this be made consistent as well?

Reply:
sure, done